docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>s/colab/Colab</p><p> </p><p>I don't think you need the "stdin" explanation. Just rephrase the second paragraph as</p><p> </p><p>"This step requires you to auth in your shell console, outside of Jupyter. In your console, execute the following command:"</p><p> </p><p>&lt;command&gt;</p><p> </p><p>As part of this flow, you will be provided with an authorization code. This code is required to consent to the Terms of Service. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
done